src/engraving/rendering/score/paint.cpp

```diff
@@ -116,6 +116,14 @@ void Paint::paintScore(Painter* painter, Score* score, const IScoreRenderer::Pai
 
             painter->beginObject("page_" + std::to_string(pi));
 
+            if (opt.isPrinting && painter->hasClipping() && (drawRect.top() < pageAbsRect.top() || drawRect.bottom() > pageAbsRect.bottom()
+                                                             || drawRect.left() < pageAbsRect.left()
+                                                             || drawRect.right() > pageAbsRect.right())) {
```

Comment:
Do we still need these `<` and `>` checks now that we're back at the `.intersected` solution?

Reply:
Technically no, but it's a minor optimisation that makes debugging and intent a bit clearer.

Reply:
I don't think so... The intent is already pretty clear, and they do nothing for optimization. It would be better to get rid of these checks